### ISSUE TITLE
Make dropdowns in nav menu open when hovered on

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/arctic-eds#106]
We can make the dropdown menus open on hover by adding the `hoverable` property, but I ran into a bug where they don't always close when you mouse out.  Restore this behavior and fix!

**Comment by cstephen** [comment]
I spent some time looking into this, and there does not appear to be a way to fix this problem without either rewriting portions of this file from the Buefy library itself:

https://github.com/buefy/buefy/blob/dev/src/components/navbar/NavbarDropdown.vue

Or coming up with a hacky JavaScript workaround from our app.

The problem is that when the mouse hovers over a hoverable `<navbar-item>`, the `is-hoverable` class is added to the element. If you then click the `<navbar-item>` (any number of times), the `is-hoverable` class is not removed from the element. Buefy's CSS is set up to expand the `<navbar-item>` sub-items when the `is-hoverable` is present and the `<navbar-item>` has focus. Clicking the `<navbar-item>` repeatedly doesn't change anything because the `is-hoverable` class stays present no matter what and focus remains on `<navbar-item>` as long as you continue clicking it.

Basically, the logic of https://github.com/buefy/buefy/blob/dev/src/components/navbar/NavbarDropdown.vue doesn't work the way that we expect.

**Comment by brucecrevensten** [comment]
There's no dropdowns, so there's no problem!
